### DISCUSSION TITLE
Inbound Secure Share

### DISCUSSION THREAD

**Discussion by giles-c** [Snowflake-Labs/terraform-provider-snowflake#516] [Q&A]
Hi,
Is it possible to have Terraform manage inbound secure shares?
A data provider has provided us with a share (https://docs.snowflake.com/en/user-guide/data-share-consumers.html) and this is landed in our account as a database by running: 
CREATE DATABASE <name> FROM SHARE <provider_account>.<share_name>

Can this be done within TF using the provider so we can manage all the RBAC still within TF as we do at the moment?
Thanks,
Giles

**Comment by serge-bouschet-cko** [comment]
As documented here: https://registry.terraform.io/providers/chanzuckerberg/snowflake/latest/docs/resources/database

`from_share` is a parameter that should cover this. I've not had much success with it so far. Keep us updated if you manage to make progress.
